CoreUtilities/test/unit/src/au/gov/asd/tac/constellation/utilities/threadpool/ConstellationGlobalThreadPoolNGTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2010-2022 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.utilities.threadpool;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeoutException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.testfx.api.FxToolkit;
+import static org.testng.Assert.*;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author altair1673
+ */
+public class ConstellationGlobalThreadPoolNGTest {
+
+    private static final Logger LOGGER = Logger.getLogger(ConstellationGlobalThreadPoolNGTest.class.getName());
+
+    public ConstellationGlobalThreadPoolNGTest() {
+    }
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        if (!FxToolkit.isFXApplicationThreadRunning()) {
+            FxToolkit.registerPrimaryStage();
+        }
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+        try {
+            FxToolkit.cleanupStages();
+        } catch (TimeoutException ex) {
+            LOGGER.log(Level.WARNING, "FxToolkit timed out trying to cleanup stages", ex);
+        }
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+    }
+
+    @AfterMethod
+    public void tearDownMethod() throws Exception {
+    }
+
+    /**
+     * Test of getThreadPool method, of class ConstellationGlobalThreadPool.
+     */
+    @Test
+    public void testGetThreadPool() {
+        System.out.println("getThreadPool");
+        assertEquals(ConstellationGlobalThreadPool.getThreadPool() instanceof ConstellationGlobalThreadPool, true);
+    }
```

Comment:
Is there a way you can add to this test (and others) to demonstrate that the object you get back is what you're expecting (i.e. some asserts about the stuff in the object itself)? e.g. Perhaps demonstrating that you get the same object back if you call it a second time?

Acknowledge that there isn't much to those functions that you can really test but at present, the test is only really checking that what you returned matched the return type of the function...which doesn't really demonstrate anything about the function (if the return type didn't match, you'd get compilation errors)